hadoop-ozone/insight/src/main/java/org/apache/hadoop/ozone/insight/ConfigurationSubCommand.java

```diff
@@ -51,16 +51,19 @@ public Void call() throws Exception {
         "Configuration for `" + insightName + "` (" + insight.getDescription()
             + ")");
     System.out.println();
-    for (Class clazz : insight.getConfigurationClasses()) {
-      showConfig(clazz);
 
+    Type type;
```

Comment:
Would you like to combine L55 and L56? I think it would be better.

Reply:
Thanks @maobaolong for this.
Fixed.